src/AuthLDAP.php

```diff
@@ -4370,4 +4382,26 @@ final public static function buildError($ds, string $message): string
         );
         return $message;
     }
+
+    private function removeDefaultFromOtherItems(int $authldaps_id): void
+    {
+        /** @var \DBmysql $DB */
+        global $DB;
+
+        if (isset($this->fields['is_default']) && (int) $this->fields["is_default"] === 1) {
+            // remove from authmails tables
+            $DB->update(
+                static::getTable(),
+                ['is_default' => 0],
+                ['id' => ['<>', $authldaps_id]]
+            );
+
+            // remove from AuthMail table
+            $DB->update(
+                AuthMail::getTable(),
+                ['is_default' => 0],
+                ['id' => ['!=', '-1']] // @todo maybe there is a better way to do this! ?
```

Comment:
If you need a `true` criteria, you can use this:
```suggestion
                [new QueryExpression('true')],
```